packages/fxa-auth-server/docs/swagger/misc-api.ts

```diff
@@ -11,7 +11,13 @@ const TAGS_MISC = {
 const ACCOUNT_GET = {
   ...TAGS_MISC,
   description: '/account',
-  notes: ['🔒 Authenticated with session token'],
+  notes: [
+    dedent`
+      🔒 Authenticated with session token
+
+      Returns the account.
```

Comment:
On further inspection, looks like it might get used by FxA Settings via the GraphQL API/fxa-auth-client? I see some references to it there, anyway. That code has always been a little opaque to me.

Reply:
@biancadanforth Heh I vaguely remember that PR. I'd just joined the team and barely knew what was going on. 😆 I do remember adding subscription `plan_names` to the delete account page but we're not using it right now to display those, but I believe we'll want to in the future. We do query this for Settings right now, and it's only used AFAICT to [conditionally render the subscriptions link](https://github.com/mozilla/fxa/blob/3045b6f2f44221117c2f264ba6494f95bbd43cc0/packages/fxa-settings/src/components/Settings/Nav/index.tsx#L16) in the nav bar.

 I see the same you're seeing on the auth client / GQL API side, [referenced here](https://github.com/mozilla/fxa/blob/3045b6f2f44221117c2f264ba6494f95bbd43cc0/packages/fxa-graphql-api/src/gql/account.resolver.ts#L767).

FWIW we query subscriptions in the admin-panel too, but I know that's in `fxa-admin-server` and I think it's done differently but I'd need to dig to find it.